src/app/books/book-list/book-list.component.ts

```diff
@@ -12,14 +12,15 @@ import { BookListItemComponent } from '../book-list-item/book-list-item.componen
   styleUrls: ['./book-list.component.css'],
   standalone: true,
   imports: [
-    NgIf, NgFor, AsyncPipe,
+    AsyncPipe,
     BookListItemComponent
   ]
 })
 export class BookListComponent {
   books$: Observable<Book[]>;
+  private service = inject(BookStoreService)
 
-  constructor(private service: BookStoreService) {
+  constructor() {
     this.books$ = this.service.getAll();
   }
```

Comment:
Das würde ich persönlich inzwischen sogar so machen, wenn ich den Service sowieso nur einmal brauche:

```ts
books$ = inject(BookStoreService).getAll();
```

Reply:
Mir fehlt in der ganzen Geschichte übrigens noch `toSignal()`. Das wäre hier ein netter Usecase, dann entfällt die AsyncPipe im Template komplett:

```ts
books = toSignal(inject(BookStoreService).getAll());
```

Reply:
Sehr guter Punkt!